test/runtests.cmd

```diff
@@ -323,15 +323,10 @@ goto :main
     set _DIRS=-all
   )
 
-  set _BuildArchMapped=%_BuildArch%
   set _BuildTypeMapped=%_BuildType%
 
-  :: Map new build arch and type names to old names until rl test tags are
-  :: updated to the new names
-  if "%_BuildArchMapped%" == "x64" set _BuildArchMapped=amd64

```

Comment:
I think we should keep the map, but reverse it to ease up transition. At a later time we'll remove it.